bin/build-sprite-string.js

```diff
@@ -0,0 +1,26 @@
+import defaultAttrs from '../src/default-attrs.json';
+
+/**
+ * Renders a SVG symbol tag
+ * @param {string} name The name of the icon
+ * @param {string} contents The contents of the icon
+ * @returns {string} the rendered SVG symbol
+ */
+function toSvgSymbol(name, contents) {
+  return `    <symbol id="${name}" viewBox="${defaultAttrs.viewBox}">
+      ${contents}
+    </symbol>\n`;
+}
+
+const svgStartTag = `<svg xmlns="${defaultAttrs.xmlns}">
+  <defs>\n`;
+
+const svgEndTag = ' </defs>\n</svg>';
+
+export default function buildSprite(icons) {
```

Comment:
Since this file is called `build-sprite-string.js`, I think it makes sense to call this function `buildSpriteString`.

Reply:
yeah, sorry!